daemon/daemon_linux_test.go

```diff
@@ -157,3 +157,10 @@ func TestTmpfsDevShmSizeOverride(t *testing.T) {
 		t.Fatal("/dev/shm not found in spec, or size option missing")
 	}
 }
+
+func TestValidateContainerIsolation(t *testing.T) {
+	d := Daemon{}
+
+	_, err := d.verifyContainerSettings("linux", &containertypes.HostConfig{Isolation: containertypes.IsolationHyperV}, nil, false)
+	assert.EqualError(t, err, "invalid isolation 'hyperv' on this daemon")
```

Comment:
Can you add separate tests for Linux and Windows (if no such test is in place yet)?

Reply:
Positive cases are more difficult to do (as it requires the daemon to have quite a lot of config values set). However, I'll add a universally failing test (both on Linux and Windows)